fv3core/stencils/c_sw.py

```diff
@@ -656,11 +657,11 @@ def __call__(
             dt2,
         )
 
-        self._fill_corners_x_delp_pt_w_stencil(delp, pt, w)
+        self._fill_corners_x_delp_pt_w_stencil(delp, pt, w, delp, pt, w)
```

Comment:
Why *are* we doing aliasing here?

Reply:
This is to circumvent a validation error thrown by the GTC toolchain,
```Illegal write and read with horizontal offset detected```
We know we can disregard this error because the regions will not overlap, but the toolchain does not know that. I will be adding TODO comments as @twicki  suggests. Eventually we will resolve these issues with compiler passes rather than frontend changes.